host/integrationbase.go

```diff
@@ -97,6 +98,16 @@ func (s *IntegrationBase) setupSuite(defaultClusterConfigFile string) {
 		Name:                                   &s.foreignDomainName,
 		WorkflowExecutionRetentionPeriodInDays: &retentionDays,
 	}))
+
+	retentionDays = 0
+	archivalStatus := workflow.ArchivalStatusEnabled
+	s.archivalDomainName = s.randomizeStr("integration-archival-enabled-domain")
+	s.Require().NoError(s.testCluster.GetFrontendClient().RegisterDomain(createContext(), &workflow.RegisterDomainRequest{
+		Name:                                   &s.archivalDomainName,
```

Comment:
can we write these in line using the utility functions common.StringPtr() and others like this?

Reply:
Oh, I thought we do not have one for archivalStatus. Will update.